tap_hubspot/streams.py

```diff
@@ -39,8 +41,10 @@ class CallsStream(HubspotStream):
     def get_url_params(
         self, context: Optional[dict], next_page_token: Optional[Any]
     ) -> Dict[str, Any]:
+        selected_properties = super().get_selected_properties()
+        hub_properties = list(set(self.properties).intersection(selected_properties))
```

Comment:
since we always do this intersection we may as well do it in `get_selected_properties`

also I have some idea you should be able to call it with `self` rather than `super()` 🤔

Reply:
yeah my very basic understanding of python is that I'm not sure super() should be used that often?

Reply:
Good shout! I've tried again in 8fd634e4904da352e17ecc1f31b7bb2ff132b1b0